addons/mail/static/src/models/thread_view_topbar.js

```diff
@@ -449,6 +449,13 @@ registerModel({
                 this.thread.isDescriptionEditableByCurrentUser
             );
         },
+        /**
+         * @private
+         * @returns {Boolean}
+         */
+        _computeShouldDisplayDebugIcon() {
+            return Boolean(odoo.debug);
```

Comment:
Not a big deal, but I'd prefer something like `odoo.debug !== ''`, I feel like it's more transparent as to what we expect `odoo.debug` to contain.